### PR TITLE
Notify Finance Dev Team on Identity, Identity Admin, Verify and Identity Infrastructure PRs

### PR DESCRIPTION
**Pull request by agnslo**
## Context

The Identification Stream from the Identity Team has merged with the Money Out team.

We want to notify the Finance Dev Team on [Identity](https://github.com/envato/identity), [Identity Admin](https://github.com/envato/identity-admin), [Verify](https://github.com/envato/verify) and [Identity Infrastructure](https://github.com/envato/identity-infrastructure) PRs in the `#finance-dev-prs` Slack channel.

As confirmed in the comments below, we no longer need to notify the Identity Team on [Identity](https://github.com/envato/identity), [Identity Admin](https://github.com/envato/identity-admin) and [Verify](https://github.com/envato/verify) PRs.

### CODE REVIEW

**Review by johnsyweb — COMMENTED** [review]
I think these repositories should be moved, rather than copied, from the Identity team.

`identity-infrastructure` will be shared for some time, so this probably belongs in both teams.

**Review by Lupeman — APPROVED** [review]
Thanks for doing this! Approved once PeteJ comment is addressed.

**Comment by agnslo** [comment]
Thanks for your comment @johnsyweb. 

I just want to confirm the following:

- We no longer want to notify Identity Team on `Identity`, `Identity Admin` and `Verify` PRs
- We only want to notify the Finance Dev Team on `Identity`, `Identity Admin` and `Verify` PRs

If that's correct, then I'll remove `identitty`, `identity-admin` and `verify` repositories from the list [here](https://github.com/envato/seal/blob/master/config/envato.yml#L99).

**Comment by johnsyweb** [comment]
Exactly this. Thanks Agnes.

**Comment by agnslo** [comment]
Thanks for confirming @johnsyweb 👍 Addressed via 9f1f6d8

**Review by mjward — APPROVED** [review]

**Review by Domon — APPROVED** [review]

File: config/envato.yml [hunk 1]
```diff
@@ -64,6 +64,8 @@ finance-dev:
     - finance-dev
     - finance_infrastructure
     - elements-earnings
+    - identity
+    - identity-admin
```
Comment (johnsyweb) [inline config/envato.yml:68]: `identity-infrastructure` will be shared for some time, so this probably belongs in both teams.

```suggestion
    - identity
    - identity-admin
    - identity-infrastructure
```

**Review by johnsyweb — COMMENTED** [review]
Thanks. One more suggestion.

File: config/envato.yml (hunk 1 above)
Comment (agnslo) [reply config/envato.yml:68]: Thanks Pete! Addressed via 1cdcdee

**Review by johnsyweb — APPROVED** [review]
🙇

**Pull request merged**